main.tf

```diff
@@ -35,7 +35,12 @@ resource "aws_transfer_server" "default" {
   identity_provider_type = "SERVICE_MANAGED"
   logging_role           = aws_iam_role.default.arn
   tags                   = var.tags
-  depends_on             = [aws_iam_role.default]
+  endpoint_type          = var.endpoint_type
+
+  endpoint_details {
+    vpc_endpoint_id = var.vpc_endpoint_id
+  }
+  depends_on = [aws_iam_role.default]
```

Comment:
Can we move this line above `endpoint_details` so it's with the other fields please?

`depends_upon` should be the last field in the group of fields.